packages/imperative/src/operations/src/Operation.ts

```diff
@@ -475,11 +474,7 @@ export abstract class Operation<T> implements ITaskWithStatus {
         for (let x = 0; x < order.length; x++) {
             this.log.info("Cleaning file: " + this.fileToUndo[x]);
             try {
-                if (fs.statSync(order[x]).isDirectory()) {
-                    removeSync(order[x]);
-                } else {
-                    fs.unlinkSync(order[x]);
-                }
+                fs.rmSync(order[x]);
```

Comment:
To keep the same behavior as before, should we pass the options `{ recursive: true, force: true }` in case the item is a directory?

I'm not sure if this matters much since I believe the `Operations` class is dead code that we may plan to remove in V4.

Reply:
Sure, will make that change.